hadoop-hdds/docs/content/feature/SCM-HA.md

```diff
@@ -91,28 +91,45 @@ For reliable HA support choose 3 independent nodes to form a quorum.
 
 ## Bootstrap
 
-The initialization of the **first** SCM-HA node is the same as a none-HA SCM:
+The initialization of the **first** SCM-HA node is the same as a non-HA SCM:
 
 ```
-bin/ozone scm --init
+ozone scm --init
 ```
 
 Second and third nodes should be *bootstrapped* instead of init. These clusters will join to the configured RAFT quorum. The id of the current server is identified by DNS name or can be set explicitly by `ozone.scm.node.id`. Most of the time you don't need to set it as DNS based id detection can work well.
 
 ```
-bin/ozone scm --bootstrap
+ozone scm --bootstrap
 ```
 
+Note: both commands perform one-time initialization.  SCM still needs to be started by running `ozone scm`.
```

Comment:
I would suggest use `ozone scm --daemon start` here, and line 131.

Reply:
Thanks for the suggestion, updated.